sysconfig-formula/_modules/suse_sysconfig.py

```diff
@@ -42,7 +42,7 @@ def fillup_regex(fillup, header_pillar=None, pattern_type=None):
     fillup_header = seek_read(fillup_template, 100, 0).decode().split('\n')[0] + '\n'
   else:
     fillup_header = ''
-  salt_header = __pillar__.get(header_pillar, '# Managed by Salt')  # noqa F821
+  salt_header = __pillar__.get(header_pillar, '# Managed by Salt\n')  # noqa F821
```

Comment:
Just wondering - if the header is provided via header_pillar, do we need to also add `\n`?

Reply:
Our custom headers are defined with the "pipe" YAML syntax, where no new lines are stripped:

```
salt-call --out=raw --local pillar.get managed_by_salt_formula
{'local': '##################################################\n##          THIS FILE IS MANAGED BY SALT        ##\n##################################################\n# If you would like to change configuration values\n# set by this file, please submit a merge request\n# modifying the pillar to our Salt repository:\n# https://gitlab.infra.opensuse.org/infra/salt\n#\n# If you would like to contribute to the Salt\n# states or templates generating this file, please\n# visit the Salt formulas repository:\n# https://github.com/openSUSE/salt-formulas\n#\n'
```

Reply:
Ok, so it won't bite us - but it might still bite other people.

I know blindly adding \n will cause an empty line in our case, but I'd still prefer that over "providing" a pitfall to other users of sysconfig-formula.

Reply:
Better?

Reply:
Yes :-) - thanks!